x-pack/plugins/security/server/authentication/api_keys/api_keys.ts

```diff
@@ -139,6 +140,31 @@ export interface InvalidateAPIKeyResult {
   }>;
 }
 
+/**
+ * Represents the parameters for validating API Key credentials.
+ */
+export interface ValidateAPIKeyParams {
+  /**
+   * Unique id for this API key
+   */
+  id: string;
+
+  /**
+   * Generated API Key (secret)
+   */
+  api_key: string;
+}
+
+/**
+ * The return value for API Key Validation
+ */
+export interface ValidateAPIKeyResult {
+  /**
+   * Indicates if the provided API Key was successfully authenticated.
+   */
+  isValid: boolean;
+}
+
```

Comment:
__note:__ It looks like we can just return `boolean` and simplify the API. Would you mind switching to `boolean` unless there is a compelling reason to use `{isValid: boolean}` object?